response_operations_ui/views/collection_exercise.py

```diff
@@ -69,15 +68,22 @@ def view_collection_exercise(short_name, period, error=None, success_message=Non
     ce_details['collection_exercise']['state'] = map_collection_exercise_state(ce_state)  # NOQA
     _format_ci_file_name(ce_details['collection_instruments'], ce_details['survey'])
 
-    if show_msg is None:
-        show_msg = request.args.get('show_msg')
+    events = {'mps', 'go_live', 'return_by', 'exercise_end'}
+    event_keys = set(formatted_events.keys())
+    if events.difference(event_keys):  # difference will be truthy if any of events are not in _keys
+        editable_events = False
+    else:
+        editable_events = True  # all expected keys are present
+
+    show_msg = request.args.get('show_msg')
+    success_panel = request.args.get('success_panel')
+    error = request.args.get('error')
```

Comment:
`error` is now never used. I expect this is needed for the [error box](https://github.com/ONSdigital/response-operations-ui/blob/master/response_operations_ui/templates/partials/error-box.html) partial by passing it into `render_template` below.

Reply:
I am going to speak to Nic about this as it appears this may have been messed up somewhere along the line.